config/search/config.json

```diff
@@ -0,0 +1,182 @@
+{
+  "automatic_index_updates": true,
+  "persist_indexes": true,
+  "number_of_threads": 8,
+  "eager_indexing_groups": [
+    [
+      {
+        "name": "public",
+        "variables": []
+      }
+    ],
+    [
+      {
+        "name": "clean",
+        "variables": []
+      }
+    ]
+  ],
+  "default_settings": {
+    "analysis": {
+      "normalizer": {
+        "custom_sort_normalizer": {
+          "type": "custom",
+          "char_filter": [],
+          "filter": [
+            "lowercase",
+            "trim",
+            "asciifolding"
+          ]
+        }
+      }
+    }
+  },
+  "types": [
+    {
+      "type": "agenda-item",
+      "on_path": "agenda-items",
+      "rdf_type": [
+        "http://data.vlaanderen.be/ns/besluit#Agendapunt"
+      ],
+      "properties": {
+        "location_id": [
+          "^http://data.vlaanderen.be/ns/besluit#behandelt",
+          "http://data.vlaanderen.be/ns/besluit#isGehoudenDoor",
+          "http://data.vlaanderen.be/ns/mandaat#isTijdspecialisatieVan",
+          "http://data.vlaanderen.be/ns/besluit#bestuurt",
+          "http://data.vlaanderen.be/ns/besluit#werkingsgebied",
+          "http://mu.semte.ch/vocabularies/core/uuid"
+        ],
+        "time_specialization_location_name": [
+          "^http://data.vlaanderen.be/ns/besluit#behandelt",
+          "http://data.vlaanderen.be/ns/besluit#isGehoudenDoor",
+          "http://data.vlaanderen.be/ns/mandaat#isTijdspecialisatieVan",
+          "http://data.vlaanderen.be/ns/besluit#bestuurt",
+          "http://data.vlaanderen.be/ns/besluit#werkingsgebied",
+          "http://www.w3.org/2000/01/rdf-schema#label"
+        ],
+        "governing_body_location_name": [
+          "^http://data.vlaanderen.be/ns/besluit#behandelt",
+          "http://data.vlaanderen.be/ns/besluit#isGehoudenDoor",
+          "http://data.vlaanderen.be/ns/besluit#bestuurt",
+          "http://data.vlaanderen.be/ns/besluit#werkingsgebied",
+          "http://www.w3.org/2000/01/rdf-schema#label"
+        ],
```

Comment:
**thought:** @nbittich What about combines `time_specialization_location_name` and `governing_body_location_name` in `location_name`? 

Should it be possible with [Composite types](https://github.com/mu-semtech/mu-search#experimental-composite-types)?

Reply:
Composite type won't merge them into one (e.g if one is null than it takes the other one), also composite types multiply indexing time by at least 3x 😬

Reply:
too bad, it would simplify the frontend life ^^